docs/release-notes/release-notes-1.7.0.md

```diff
@@ -0,0 +1,1434 @@
+consensus changes is already included in this release, however it will remain
+dormant until the stakeholders vote to activate it.
+
+For reference, the consensus change work for each of the four changes was
+originally proposed and approved for initial implementation via the following
+Politeia proposals:
+- [Decentralized Treasury Spending](https://proposals-archive.decred.org/proposals/c96290a)
+- [Explicit Version Upgrades Consensus Change](https://proposals.decred.org/record/3a98861)
+- [Automatic Ticket Revocations Consensus Change](https://proposals.decred.org/record/e2d7b7d)
+- [Change PoW/PoS Subsidy Split From 60/30 to 10/80](https://proposals.decred.org/record/427e1d4)
+
+The following Decred Change Proposals (DCPs) describe the proposed changes in
+detail and provide full technical specifications:
+- [DCP0007](https://github.com/decred/dcps/blob/master/dcp-0007/dcp-0007.mediawiki)
+- [DCP0008](https://github.com/decred/dcps/blob/master/dcp-0008/dcp-0008.mediawiki)
+- [DCP0009](https://github.com/decred/dcps/blob/master/dcp-0009/dcp-0009.mediawiki)
+- [DCP0010](https://github.com/decred/dcps/blob/master/dcp-0010/dcp-0010.mediawiki)
+
+**It is important for everyone to upgrade their software to this latest release
+even if you don't intend to vote in favor of the agenda.**
+
+## Downgrade Warning
+
+The database format in v1.7.0 is not compatible with previous versions of the
+software.  This only affects downgrades as users upgrading from previous
+versions will see a one time database migration.
+
+Once this migration has been completed, it will no longer be possible to
+downgrade to a previous version of the software without having to delete the
+database and redownload the chain.
+
+The database migration typically takes around 40-50 minutes on HDDs and 20-30
+minutes on SSDs.
+
+## Notable Changes
+
+### Four New Consensus Change Votes
+
+Four new consensus change votes are now available as of this release.  After
+upgrading, stakeholders may set their preferences through their wallet.
+
+### Revert Treasury Maximum Expenditure Policy Vote
+
+The first new vote available as of this release has the id `reverttreasurypolicy`.
+
+The primary goal of this change is to revert the currently active maximum
+expenditure policy of the decentralized Treasury to the one specified in the
+[original Politeia proposal](https://proposals-archive.decred.org/proposals/c96290a).
+
+See [DCP0007](https://github.com/decred/dcps/blob/master/dcp-0007/dcp-0007.mediawiki) for
+the full technical specification.
+
+### Explicit Version Upgrades Vote
+
+The second new vote available as of this release has the id `explicitverupgrades`.
+
+The primary goals of this change are to:
+
+* Provide an easy, reliable, and efficient method for software and hardware to
+  determine exactly which rules should be applied to transaction and script
+  versions
+* Further embrace the increased security, and other desirable properties, that
```

Comment:
I don't think this sentence needs any commas.

Reply:
Updated.